### ISSUE TITLE
Linux: closing game leaves "defunct" processes that need to be cleaned up manually

### ISSUE THREAD

**Issue by rhmkrutov** [qrrk/Catapult#18]
Not sure what else to describe.

**Comment by qrrk** [comment]
Yes, this is a bug in Godot on Linux. I remember seeing the ticket, but can't find it right now. The zombie processes are cleaned up when the launcher exits, so it's not a huge issue. I hope it will be fixed eventually.
